lib/phlex/compiler/method_compiler.rb

```diff
@@ -194,6 +198,19 @@ def compile_fragment_helper_block(node)
 			)
 		end
 
+		def compile_comment_helper(node)
+			[
+				buffer("<!-- "),
+				visit_phlex_block(node.block),
```

Comment:
We may be able to just visit the block body here so it doesn't split the buffer.